app/display.c

```diff
@@ -143,10 +144,14 @@ void display_init(void)
     clear_screen_region(ROW_FOOTER, 0, ROW_FOOTER, SCREEN_WIDTH - 1);
     prints(ROW_FOOTER, 0, " <ESC> Exit  <F1> Configuration  <Space> Scroll Lock");
     prints(ROW_FOOTER, 64, MT_VERSION "." GIT_HASH);
-#if TESTWORD_WIDTH > 32
-    prints(ROW_FOOTER, 76, ".x64");
+#ifdef CONFIG_64BIT
```

Comment:
I think that __x86_64__ / __loongarch_lp64 / __i386__ checks would be better here, no need for CONFIG_64BIT.

Reply:
OK, I will removed the `CONFIG_64BIT` next time.